NOpenCL/Device.cs

```diff
@@ -10,14 +10,26 @@ namespace NOpenCL
     using System.ComponentModel;
     using NOpenCL.SafeHandles;
     using DeviceInfo = NOpenCL.UnsafeNativeMethods.DeviceInfo;
+    using System.Diagnostics;
 
+    [DebuggerDisplay("Name = {Name}")]
```

Comment:
:bulb: See my note/questions on the previous use of `[DebuggerDisplay]`. If you make a separate pull request, this use of DebuggerDisplay would be appropriate to include at the same time as the other.